### ISSUE TITLE
Loss and Accuracy are reversed

### ISSUE THREAD

**Issue by ToshikiShimizu** [tugstugi/pytorch-speech-commands#4]
https://github.com/tugstugi/pytorch-speech-commands/blob/master/train_speech_commands.py
L249-256
```
    if accuracy > best_accuracy:
        best_accuracy = accuracy
        torch.save(checkpoint, 'checkpoints/best-loss-speech-commands-checkpoint-%s.pth' % full_name)
        torch.save(model, '%d-%s-best-loss.pth' % (start_timestamp, full_name))
    if epoch_loss < best_loss:
        best_loss = epoch_loss
        torch.save(checkpoint, 'checkpoints/best-acc-speech-commands-checkpoint-%s.pth' % full_name)
        torch.save(model, '%d-%s-best-acc.pth' % (start_timestamp, full_name))
```

**Comment by tugstugi** [comment]
good catch, but I will not fix it because the project is not maintained anymore :)
